guides/plugins/apps/administration/add-custom-modules.md

```diff
@@ -9,17 +9,17 @@ Creating custom modules takes place at the `<admin>` section of your `manifest.x
 To configure your module you can set it up with with some additional attributes.
 
 * `name` \(required\): The technical name of the module. This is the name your module is referenced with.
+* `parent` \(required\): The Administration navigation id from the menu item that serves as the parent menu item. If omitted your module will be listed under the "My apps" menu entry.
```

Comment:
```suggestion
* `parent` \(required\): The Administration navigation id of the menu item that serves as the parent menu item.
```

It can't be omitted anymore 😄